docs/topics/mpp/mpp-add-dependencies.md

```diff
@@ -1,7 +1,7 @@
 [//]: # (title: Add dependencies)
 
 To add a dependency on a library, set the dependency of the required [type](gradle.md#dependency-types) (for example, `implementation`) in the 
-`dependencies` block of the source sets DSL.
+`dependencies` block in your [Gradle](gradle.md) build script.
```

Comment:
TBH, I have a concern about `in` doubling in this sentence 🤔

Reply:
Thank you, I will ask for a proofread.

Reply:
Wouldn't this link be better here? https://kotlinlang.org/docs/mpp-dsl-reference.html#dependencies
It's MPP- and dependencies-specific, as opposed to just gradle.md.

Reply:
Thank you, Pasha! I've got an idea – what if to leave the link to Gradle, but also add the link you proposed? See the result in the diff.

Regarding the proofread: there is a couple of minor changes, I put them in the code too, but proofreaders didn't have any concerns regarding doubling `in`.